src/registrar/templates/includes/profile_information.html

```diff
@@ -0,0 +1,36 @@
+{% load static field_helpers %}
+
+{% block domain_content %} 
+
+<!-- {% if not user.has_contact_info %}
+<div class="usa-alert usa-alert--error margin-bottom-2">
+  <div class="usa-alert__body">
+    <h4 class="usa-alert__heading larger-font-sizing">
+      Missing contact information</h4>
+    <p class="usa-alert__text ">
+      We appear to be missing some contact information from you.  Please visit your profile.
+    </p>
+  </div>
+</div>
+{% endif %} -->
+<div
+  class="usa-summary-box"
+  role="region"
+  aria-labelledby="summary-box-key-information"
+>
+  <div class="usa-summary-box__body">
+    <h3 class="usa-summary-box__heading" id="summary-box-key-information">
+      Your contact information
+    </h3>
+    <div class="usa-summary-box__text">
+      <ul>
+        <li>Full name: <b>{{ user.contact.first_name }} {{ user.contact.last_name }}</b></li>
```

Comment:
I prefer the use of `<strong>` instead of `<b>`. It's the difference between bolding and emphasizing.

Reply:
Good thought.  It's unclear whether the design wants us to simply emphasize, though, so I went with a literal translation so there is no question that this will be bold, like the design shows.  This is a good question to bring up with design, though.